tools/flang1/flang1exe/lowersym.c

```diff
@@ -2715,6 +2715,15 @@ lower_put_datatype(int dtype, int usage)
         }
       }
     }
+
+    /* If align pragma value is smaller than its original alignment,
+     * then align pragma should have no effect
+     */
+    if (DTA(dtype) == alignment(dtype))
```

Comment:
`==` &rarr; `>` ?

Reply:
> `==` → `>` ?

Yes, you are correct. I have corrected the code using >.